### PR TITLE
VB-1398: Remove feedback banner/move beta tag

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

**Review by hutcheonb-moj — APPROVED** [review]
LGTM

File: server/views/partials/header.njk [hunk 1]
```diff
@@ -35,17 +35,12 @@
...
-  <span class="location-bar__location" data-test="active-location">
-    Hewell (HMP)
-  </span>
   {% if not hidePhaseBanner %}
-    {{ govukPhaseBanner ({
-      tag: {
-        text: "beta"
-      },
-       html: 'This is a new service, <a class="govuk-link" target="_blank" href="https://eu.surveymonkey.com/r/9ZBN3YJ">give us your feedback (opens in a new tab)</a>',
-      classes: 'bapv-phase-banner'
```
Comment (tpmcgowan) [inline server/views/partials/header.njk:47]: If this is going, can we clean up the `.bapv-phase-banner` stuff from `local.scss`?

Comment (psoleckimoj) [reply server/views/partials/header.njk:47]: @tpmcgowan Sorry only just saw this!

Comment (psoleckimoj) [reply server/views/partials/header.njk:47]: @tpmcgowan PR raised https://github.com/ministryofjustice/book-a-prison-visit-staff-ui/pull/402

**Pull request merged**